client/asset/btc/btc.go

```diff
@@ -45,21 +44,61 @@ const (
 	BipID                   = 0
 	// The default fee is passed to the user as part of the asset.WalletInfo
 	// structure.
-	defaultFee         = 112
+	defaultFee         = 100
 	minNetworkVersion  = 190000
 	minProtocolVersion = 70015
 )
 
 var (
 	// blockTicker is the delay between calls to check for new blocks.
-	blockTicker = time.Second
+	blockTicker    = time.Second
+	fallbackFeeKey = "fallbackfee"
+	configOpts     = []*asset.ConfigOption{
+		{
+			Key:         "walletname",
+			DisplayName: "Wallet Name",
+			Description: "The wallet name",
+		},
+		{
+			Key:         "rpcuser",
+			DisplayName: "JSON-RPC Username",
+			Description: "Bitcoin's 'rpcuser' setting",
+		},
+		{
+			Key:         "rpcpassword",
+			DisplayName: "JSON-RPC Password",
+			Description: "Bitcoin's 'rpcpassword' setting",
+			NoEcho:      true,
+		},
+		{
+			Key:         "rpcbind",
+			DisplayName: "JSON-RPC Address",
+			Description: "<addr> or <addr>:<port> (default 'localhost')",
+		},
+		{
+			Key:         "rpcport",
+			DisplayName: "JSON-RPC Port",
+			Description: "Port for RPC connections (if not set in Address)",
+		},
+		{
+			Key:          fallbackFeeKey,
+			DisplayName:  "Fallback fee rate",
+			Description:  "Bitcoin's 'fallbackfee' rate. Units: Sats/kB",
```

Comment:
We're interpreting this as sats/byte presently, although we have talked about sats/kB instead.  Same in ltc.go